src/models.py

```diff
@@ -126,18 +142,28 @@ class Services(db.Model):
     # realtionships
     users_operations = db.relationship("Operations", back_populates="service_operations")
 
-    def create_user(self):
-        db.session.add(self)
-        db.session.commit()
-    # def read_user():
+    def serialize(self):
+        return {
+            "id": self.id,
+            "id_service_type": self.id_service_type,
+            "id_user_offer": self.id_user_offer,
+            "description": self.description,
+            "price_h": self.price_h
+        }
 
-    # def update_user():
+    @classmethod
+    def read_all_services(cls):
+        services = Services.query.all()
+        all_services =  list(map(lambda x: x.serialize(), services))
+        return all_services
 
-    # def delete_user():
+    @classmethod 
+    def read_user_services(cls, id_user):
+        services  = Services.query.filter_by(id_user_offer = id_user)
```

Comment:
añade .first() para solo devolver uno y optimizar

Reply:
@JimenaEB 
El .first() no me deja usarlo porque el objeto Services no es iterable

Reply:
Debería dejarte aunque no sea iterable, lo vemos juntyos si quieres
Si no también tenemos la opción de .one() pero esta opción lanza una excepción que debemos controlar
https://stackoverflow.com/questions/29161730/what-is-the-difference-between-one-and-first